daemon/daemon/swap_in.go

```diff
@@ -45,6 +47,13 @@ func (m *SwapMonitor) MonitorSwapIn(ctx context.Context, currentSwap models.Swap
 		// Do nothing
 		logger.Debug("waiting for payment")
 	case models.StatusContractFundedUnconfirmed:
+		if newSwap.LockTx != nil {
```

Comment:
I think it will always be not null

Reply:
yep, but for protection, otherwise it could crash the daemon